.vscode/tasks.json

```diff
@@ -0,0 +1,25 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+        "label": "build",
+        "command": "./build.cmd",
+        "type": "shell",
+        "args": [
+        ],
+        "problemMatcher": "$msCompile",
+        "group": "build"
+    },
+    {
+        "label": "build fast",
```

Comment:
Want to call this "build without analyzers" or something so it's obvious what it actually does?

Reply:
I assumed "build fast" was pretty clear :trollface: 

Will change to "build skipAnalyzers"